src/components/Header.tsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable @typescript-eslint/indent */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useContext, useState } from 'react';
+import { TodosContext } from '../TodosContext';
+import { USER_ID } from '../constants';
+import { createTodo } from '../api/todos';
+import { Todo } from '../types/Todo';
+
+export const getNextId = (todos: Todo[]) => {
+  const ids = todos.map((todo) => todo.id);
+
+  return Math.max(...ids) + 1;
+};
```

Comment:
You can remove this function already, bc in this task we don't need to create an id for todo